RationaleMCP/ReadMe.md

```diff
@@ -30,7 +30,7 @@ but the rest of the development on a branch/pull-request before being accepted.
 - ~~MCP0024~~ Initialization of Clocked States (added in Modelica 3.4) ([#2007](https://github.com/modelica/ModelicaSpecification/issues/2007))
 - ~~MCP0023~~ Model to Record (added in Modelica 3.4) ([#1953](https://github.com/modelica/ModelicaSpecification/issues/1953))
 - ~~MCP0022~~ Integer to Enumeration (added in Modelica 3.4) ([#1842](https://github.com/modelica/ModelicaSpecification/issues/1842))
-- MCP0021 Component iterators (active)
+- MCP0021 Component iterators ([MCP/0021](https://github.com/modelica/ModelicaSpecification/tree/MCP/0032/RationaleMCP/0021))
```

Comment:
Note that this link does not work. I believe the "0032" part should read "0021".

Reply:
Ah, yes.